src/Navigation/styled.js

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components"
+
+export const Wrapper = styled.div`
+    background-color: ${({ theme }) => theme.color.almostBlack};
+    min-height: 93px;
+    margin:0 auto;
+`;
+
+export const Container = styled.div`
+    max-width: 1300px;
+    margin:0 auto;
+    display: flex;
+    align-items: center;
```

Comment:
I would add `justify-content: space-between;` to Navigation/styled.js/Container to separate `<Header />` and `<Search />`